### DISCUSSION TITLE
24bit/192khz Throughput

### DISCUSSION THREAD

**Discussion by cledwynclegg** [mikebrady/shairport-sync#1508] [General]
Hi Mike

How do I get a 24bit/192khz throughput?

I'm using your shairport-sync built with the following instructions here: https://thepi.io/how-to-set-up-a-raspberry-pi-airplay-receiver/

My set-up is as follows:

- First Gen Raspberry PI Zero running Stretch Lite
- Just Boom Digi Zero
- Cambridge Audio Dac Magic 100 via RCA S/PDIF

The Dac Magic shows 44.1khz on all music even if it has a higher bitrate.

Is there anything I can do here? Would installing Alac help?

Thank you in advance.

Cheers
Cledwyn

**Comment by mikebrady** [comment]
Thanks for the post. AirPlay audio is 44,100 frames per second with ALAC lossless encoding. It's never been any different, irrespective of the format of the original. AirPlay 2 audio is forwarded to Shairport Sync at [44,100 samples per second](https://github.com/mikebrady/shairport-sync/blob/development/AIRPLAY2.md) no matter how it's originally encoded. We don't know how to get different formats in AirPlay 2, I'm afraid.

**Reply by cledwynclegg** [reply]
Hi Mike

Thank you for clarifying this. Anyhow, regardless of the bitrate, it still sounds excellent. This was the first time I had hooked anything up to a DAC that showed the bitrate.

Please let me know if you do figure out how to get different formats in AirPlay 2.

Cheers
Cledwyn
